src/imagestat/mod.rs

```diff
@@ -0,0 +1,70 @@
+//! Image Statistics Functions
+
+use color;
+use color::ColorType;
+use color::Rgba;
+use dynimage::DynamicImage;
+use image::GenericImage;
+use num;
+use std;
+
+#[allow(missing_copy_implementations)]
+/// Contains the values for a channel.
+pub struct HistogramChannel {
+	/// Contains the count of pixels for each value.
+	pub values: Vec<i32>,
+}
+
+#[allow(missing_copy_implementations)]
+/// Contains the tonal information for an image.
+/// Supports both color and greyscale images.
+pub struct Histogram {
+	/// The color channels for the histogram.
+	pub channels: Vec<HistogramChannel>,
+}
+
+impl Histogram {
+	/// Sets the count for a particular value of pixels.
+    pub fn set_count(&mut self, channel: usize, value: usize, count: i32)
+	{
```

Comment:
This bracket should be on the previous line. You’re also mixing tabs with spaces, please use 4 spaces everywhere.